modules/olm-configuring-aws-sts.adoc

```diff
@@ -0,0 +1,228 @@
+<2> Specify a namespace for the CR. For easier reference, choose a name related to the name of the Operator.
+<3> Specify IAM JSON policy statement entries. The values in this example are provided for formatting clarification only and must be updated for the requirements of the Operator. For more information, see AWS documentation about link:https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_statement.html[IAM JSON policy elements: Statement].
+<4> Specify a name for the secret that will be used by the Operator. For easier reference, choose a name related to the name of the Operator.
+<5> Specify the namespace in which the Operator that consumes the AWS credentials is deployed. For easier reference, choose a name related to the name of the Operator.
+<6> Specify the name of the service account that is used by the Operator.
+endif::[]
+
+. Retrieve the `serviceAccountIssuer` value for your cluster by running the following command:
++
+[source, terminal]
+----
+$ oc get authentication cluster -o jsonpath --template='{ .spec.serviceAccountIssuer }'
+----
++
+.Example output
++
+[source, terminal]
+----
+https://<cluster_name>-oidc.s3.<aws_region>.amazonaws.com <1>
+----
+<1> In this example, the issuer URL is `<cluster_name>-oidc.s3.<aws_region>.amazonaws.com`, without the `https://` prefix.
+
+. Use the `ccoctl` tool to process all `CredentialsRequest` objects in the `credrequests` directory by running the following command:
++
+[source, terminal]
+----
+$ ccoctl aws create-iam-roles \
+  --name=<name> \
+  --region=<aws_region> \
+  --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \
+  --identity-provider-arn=arn:aws:iam::<aws_account_id>:oidc-provider/<issuer_URL>
+----
++
+where:
++
+--
+** `<name>` is the name used to tag any cloud resources that are created for tracking.
+** `<aws_region>` is the AWS region in which cloud resources will be created.
+** `<path_to_ccoctl_output_dir>` is the path to the public key file that the `ccoctl aws create-key-pair` command generated.
+** `<aws_account_id>` is your AWS account ID and `<issuer_URL>` is the `serviceAccountIssuer` value for your cluster. These are standard elements of the Amazon Resource Name (ARN) for your cluster, provided here to illustrate the format of an ARN. You can obtain the ARN for your cluster's identity provider from the *Identity Providers* menu in the link:https://console.aws.amazon.com/iam/[AWS IAM console].
+--
++
+The command creates an IAM role in AWS for the Operator and a corresponding `<operator_secret_namespace>-<operator_secret_name>-credentials.yaml` file in the `<path_to_ccoctl_output_dir>/manifests/` directory of the local file system.
+
+. Optional: To verify that the manifests with cloud credentials exist, list the contents of the `<path_to_ccoctl_output_dir>/manifests` directory by running the following command:
++
```

Comment:
In format of "ccoctl aws create-iam-roles" command, parameter --credentials-requests-dir is added but no --output-dir=<path_to_ccoctl_output_dir>, 
but in the descriptions there are no <path_to_directory_with_list_of_credentials_requests> but <path_to_ccoctl_output_dir>.
Better to add --output-dir parameter and make these 2 places consistent.